### PR TITLE
Deploying pestguard

### PR DESCRIPTION
**Pull request by Faith-wanjiru**
# Description

Deployment generally refers to the process of making a software application, system, or update available for use. It involves moving the code or system from a development or staging environment to a live or production environment where it can be accessed and used by end users

Please include a summary of the change and which issue is fixed. 
Please also include relevant motivation and context. 
List any dependencies that are required for this change.

Perhaps answering the following questions will help create a good description:
* What is this?

Deployment generally refers to the process of making a software application, system, or update available for use. It involves moving the code or system from a development or staging environment to a live or production environment where it can be accessed and used by end users

* Why this?

Software Deployment: If you're talking about a software project, "deployment" refers to the process of moving the final version of the software from a development or testing environment to a live environment where end users can access it. This process ensures that the software is properly installed, configured, and functioning as intended.
IT Infrastructure Deployment: In IT, deployment might involve setting up and configuring hardware, servers, or network components. For example, deploying a new server involves physically installing it, configuring it, and ensuring it integrates properly with the existing infrastructure.

* How are you doing this? 

Build: Compile and package the code into executable files or deployable artifacts.
Testing: Ensure that the software is tested in a staging environment to catch any issues before going live.

Fixes # (issue) 

https://wanyoikefaithwanjiru-1723189027394.atlassian.net/jira/software/projects/SCRUM/boards/1/backlog?epics=visible

## Type of change

Military Deployment
Cloud Deployment
IT Infrastructure Deployment

Specify the kind of PR/MR

Please delete options that are not relevant.

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A - e.g Manually tested user creation
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Cynthia-Matsitsa — APPROVED** [review]

**Review by Esther-Naserian — APPROVED** [review]

**Review by stephanie-niva — APPROVED** [review]

File: pest_guard/pest_guard/settings.py [hunk 1]
```diff
@@ -137,6 +158,8 @@
 # https://docs.djangoproject.com/en/5.1/howto/static-files/
 
 STATIC_URL = 'static/'
+STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
+DATABASES['default'] = dj_database_url.config(conn_max_age=600, ssl_require=True)
```
Comment (Ndichu-shee) [inline pest_guard/pest_guard/settings.py:162]: remove this

File: pest_guard/pest_guard/settings.py [hunk 2]
```diff
@@ -146,5 +169,4 @@
 AUTH0_CLIENT_ID = os.environ.get("AUTH0_CLIENT_ID")
 AUTH0_CLIENT_SECRET = os.environ.get("AUTH0_CLIENT_SECRET")
 
-REDIRECT_URI = 'http://localhost:8000/auth/callback'
 REDIRECT_URI= 'http://localhost:8000/auth/'
```
Comment (Ndichu-shee) [inline pest_guard/pest_guard/settings.py:172]: add this in the env variable

File: pest_guard/pest_guard/settings.py [hunk 3]
```diff
@@ -146,5 +168,6 @@
 AUTH0_CLIENT_ID = os.environ.get("AUTH0_CLIENT_ID")
 AUTH0_CLIENT_SECRET = os.environ.get("AUTH0_CLIENT_SECRET")
 
+
 REDIRECT_URI = 'http://localhost:8000/auth/callback'
-REDIRECT_URI= 'http://localhost:8000/auth/'
\ No newline at end of file
+REDIRECT_URI= 'http://localhost:8000/auth/'
```
Comment (Ndichu-shee) [inline pest_guard/pest_guard/settings.py:173]: hide this

**Review by Ndichu-shee — APPROVED** [review]

File: pest_guard/settings.py [hunk 4]
```diff
@@ -142,9 +124,3 @@
 # https://docs.djangoproject.com/en/5.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
-AUTH0_DOMAIN = os.environ.get("AUTH0_DOMAIN")
```
Comment (Ndichu-shee) [inline pest_guard/settings.py:145]: why have you removed this

**Pull request merged**